crates/bevy_pbr/src/render/light.rs

```diff
@@ -1699,7 +1702,7 @@ pub fn queue_shadows<M: Material>(
 }
 
 pub struct Shadow {
-    pub distance: f32,
+    pub asset_id: AssetId<Mesh>,
```

Comment:
In the future we probably want to be able to represent `AssetID` as a u64 or something so we can go back to using radsort, as I noticed decreases in sort performance in the opaque sorting by pipeline PR (that were more than made up for by the better batching).

Reply:
definitely, should be easy to do.